### PR TITLE
Fixed type inconsistencies, reducing warnings when compiled.

### PR DESCRIPTION
**Pull request by alexsaves**
Comparing uint32's and size_t's to unsigned ints.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ build/
 *.dat
 *.trie
 *.log
+.idea
```
Comment (yorkie) [inline .gitignore]: why add this?

File: binding.gyp [hunk 2]
```diff
@@ -5,7 +5,7 @@
       "sources": [
         "src/snprintf/snprintf.c",
         "src/pattern/51Degrees.c",
-        "src/pattern/api.cc",
+        "src/pattern/api.cc"
```
Comment (yorkie) [inline binding.gyp]: don't make this change

File: binding.gyp [hunk 3]
```diff
@@ -15,14 +15,14 @@
       ],
       "include_dirs" : [
         "<!(node -e \"require('nan')\")"
-      ],
+      ]
```
Comment (yorkie) [inline binding.gyp]: ditto

File: binding.gyp [hunk 4]
```diff
@@ -15,14 +15,14 @@
...
     },
     {
       "target_name": "trie",
       "sources": [
         "src/snprintf/snprintf.c",
         "src/trie/51Degrees.c",
-        "src/trie/api.cc",
+        "src/trie/api.cc"
```
Comment (yorkie) [inline binding.gyp]: ditto

File: binding.gyp [hunk 5]
```diff
@@ -32,7 +32,7 @@
       ],
       "include_dirs" : [
         "<!(node -e \"require('nan')\")"
-      ],
-    },
+      ]
+    }
```
Comment (yorkie) [inline binding.gyp]: ditto

File: package.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
   "name": "51degrees",
-  "version": "1.8.1",
+  "version": "1.8.2",
   "description": "51degrees c-sdk native bindings for nodejs",
   "main": "index.js",
   "scripts": {
     "test": "NODE_ENV=test node test.js",
-    "install": "node-gyp rebuild --msvs_version=2012",
-    "build": "node-gyp configure --msvs_version=2012 && node-gyp build"
+    "install": "node-gyp rebuild",
```
Comment (yorkie) [inline package.json]: why remove the flag `--msvs_version=2012` that makes window be working

**Comment by yorkie** [comment]
First of all, thank you @alexsaves for this PR, but may I ask you to update the source only?

**Comment by alexsaves** [comment]
Sure. Im trying to get it to build consistently in linux. Its hard to imagine many production environments that would be windows based but I see your point.

**Pull request closed without merging**